src/views/_proxied-dot-io/versioned-error/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import useErrorPageAnalytics from '@hashicorp/react-error-view/use-error-page-analytics'
+import ErrorPage, { ErrorPageProps } from '@hashicorp/react-error-view'
+import s from '@hashicorp/react-error-view/style.module.css'
+
+const versionPattern = /\/(?<version>v\d+[.]\d+[.](\d+|x))/
+
+function VersionNotFound({ version }: { version: string }): React.ReactElement {
+  const { asPath } = useRouter()
+  useErrorPageAnalytics(404)
+
+  const pathWithoutVersion = asPath.replace(versionPattern, '')
+  const basePath = asPath.split('/')[1]
+
+  return (
+    <div className={s.root}>
+      <h1 className={s.heading}>
+        This page does not exist for version {version}.
+      </h1>
+      <p>
+        Please select either the{' '}
+        <Link href={pathWithoutVersion}>
+          <a>most recent version</a>
+        </Link>{' '}
+        or a valid version that includes the page you are looking for.
+      </p>
+      <p>
+        <Link href={`/${basePath}`}>
```

Comment:
I'm seeing

```html
<a href="/_proxied-dot-io">← Go back to Documentation</a>
```

Is this just a byproduct of the preview environment?

Reply:
I noticed this too, debugged it for quite awhile today 😞  When clicking on the links it takes you to the correct place. And inspecting the components in the react devtools shows the right values within the `Link` component, but for some reason the actual href attribute isn't updated.

I'm hoping it'll be correct when deployed, but if not I'll circle back.